### PR TITLE
[bitnami/magento] Chart standardised + Bump dependencies

### PR DESCRIPTION
**Pull request by migruiz4**
Signed-off-by: Miguel Ruiz <miruiz@vmware.com>

**Description of the change**

This PR standardizes the bitnami/magento chart to be in line with the rest of the catalog and updates its dependencies.

**Checklist**

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/magento/templates/pv.yaml [hunk 1]
```diff
@@ -2,19 +2,24 @@
...
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
 spec:
   accessModes:
-    - {{ .Values.persistence.accessMode | quote }}
-  capacity:
+    {{- range .Values.persistence.accessModes }}
+    - {{ . | quote }}
+    {{- end }}  capacity:
```
Comment (fmulero) [inline bitnami/magento/templates/pv.yaml:22]:
```suggestion
    {{- end }}
  capacity:
```

File: bitnami/magento/README.md [hunk 2]
```diff
@@ -632,6 +653,18 @@ You can disable the initContainer using the `elasticsearch.sysctlImage.enabled=f
...
+This major release renames several values in this chart and adds missing features, in order to get aligned with the rest of the assets in the Bitnami charts repository.
+
+Additionally, it updates both the [Elasticsearch](https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch) and the [MariaDB](https://github.com/bitnami/charts/tree/master/bitnami/mariadb) subcharts to their latest major versions, 18.0.0 and 10.0.0 respectively, where similar changes have been also performed.
+Check [Elasticsearch Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/redis#to-1800) and [Redis Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/mariadb#to-1000) for more information.
+
+The following values have been renamed:
+
+- `service.port` renamed as `service.ports.http`.
+- `service.httpsPort` renamed as `service.ports.https`.
+
```
Comment (fmulero) [inline bitnami/magento/README.md:667]: We should add that  `persistence.accessMode` is now a list

**Review by fmulero — CHANGES_REQUESTED** [review]
Hi @migruiz4, Please have a look to the following comments

**Review by fmulero — DISMISSED** [review]
LGTM

File: bitnami/magento/Chart.yaml [hunk 3]
```diff
@@ -8,11 +8,11 @@ dependencies:
     repository: https://charts.bitnami.com/bitnami
     tags:
       - magento-database
-    version: 9.x.x
+    version: 10.x.x
```
Comment (carrodher) [inline bitnami/magento/Chart.yaml:11]: Please note there is a new major in the database, see https://github.com/bitnami/charts/pull/9827
```suggestion
    version: 11.x.x
```

**Review by fmulero — APPROVED** [review]
👍

**Review by fmulero — APPROVED** [review]
Good Catch!

**Review by carrodher — DISMISSED** [review]

File: bitnami/magento/README.md [hunk 4]
```diff
@@ -632,6 +654,20 @@ You can disable the initContainer using the `elasticsearch.sysctlImage.enabled=f
 
 ## Upgrading
 
+### To 18.0.0
+
+This major release renames several values in this chart and adds missing features, in order to get aligned with the rest of the assets in the Bitnami charts repository.
+
+Additionally, it updates both the [Elasticsearch](https://github.com/bitnami/charts/tree/master/bitnami/elasticsearch) and the [MariaDB](https://github.com/bitnami/charts/tree/master/bitnami/mariadb) subcharts to their latest major versions, 18.0.0 and 10.0.0 respectively, where similar changes have been also performed.
+Check [Elasticsearch Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/redis#to-1800) and [Redis Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/mariadb#to-1000) for more information.
```
Comment (carrodher) [inline bitnami/magento/README.md:662]:
```suggestion
Check [Elasticsearch Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/redis#to-1800) and [MariaDB Upgrading Notes](https://github.com/bitnami/charts/tree/master/bitnami/mariadb#to-1000) for more information.
```

**Review by carrodher — DISMISSED** [review]

**Pull request merged**